dashboard/src/components/Config/AppRepoList/AppRepoForm.tsx

```diff
@@ -27,21 +27,24 @@ interface IAppRepoFormProps {
   kubeappsNamespace: string;
   repo?: IAppRepository;
   secret?: ISecret;
+  appVersion: string;
 }
 
 const AUTH_METHOD_NONE = "none";
 const AUTH_METHOD_BASIC = "basic";
 const AUTH_METHOD_BEARER = "bearer";
 const AUTH_METHOD_CUSTOM = "custom";
 
-export function AppRepoForm({
-  onSubmit,
-  onAfterInstall,
-  namespace,
-  kubeappsNamespace,
-  repo,
-  secret,
-}: IAppRepoFormProps) {
+export function AppRepoForm(props: IAppRepoFormProps) {
+  const {
+    onSubmit,
+    onAfterInstall,
+    namespace,
+    kubeappsNamespace,
+    repo,
+    secret,
+    appVersion,
```

Comment:
Rather than forwarding the `appVersion` around the different components, you can just get it from the state (see the `useSelector` method at line 65 in this file). You can use `useSelector` to have access to the application state so just need to change that:

```js
const {
  repos: {
    imagePullSecrets,
    errors: { create: createError, update: updateError, validate: validationError },
    validating,
  },
  config: {
    appVersion,
  },
  } = useSelector((state: IStoreState) => state);
```

Also, you may need to update the tests for this file since the partial `state` defined there won't contain the `config` section.

Reply:
Much easier!! I'm not very used to react state yet, so I simply wasn't aware of this feature. Thanks for pointing me out!